src/Utils/SchemaExtender.php

```diff
@@ -314,45 +319,43 @@ protected static function extendType($typeDef)
             return Type::nonNull(static::extendType($typeDef->getWrappedType()));
         }
 
+        // @phpstan-ignore-next-line generic is not properly caught
```

Comment:
if this is not intentionally caught, we should update baseline instead. Phpstan will then throw error when it's caught and the baseline entry is redundant